test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/RunnableProjectGeneratorTests.cs

```diff
@@ -1,48 +0,0 @@
-using Microsoft.TemplateEngine.Abstractions;
```

Comment:
why it is removed?

Reply:
Because it was testing `ConfigTimestampUtc` which is now removed.